src/Composer/Plugin.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\ManagerPlugin\Composer;
+
+use Composer\Composer;
+use Composer\EventDispatcher\EventSubscriberInterface;
+use Composer\IO\IOInterface;
+use Composer\Plugin\PluginInterface;
+use Composer\Script\Event;
+use Composer\Script\ScriptEvents;
+
+class Plugin implements PluginInterface, EventSubscriberInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function activate(Composer $composer, IOInterface $io): void
+    {
+        // Nothing to do here, as all features are provided through event listeners
+    }
+
+    /**
+     * Dumps the Contao Manager plugins.
+     *
+     * @param Event $event
+     */
+    public function dumpPlugins(Event $event): void
+    {
+        $composer = $event->getComposer();
+
+        $installer = new Installer();
+        $installer->dumpPlugins($composer->getLocker());
+
+        $event->getIO()->write('Dumped Contao Manager plugins');
```

Comment:
Maybe make the output a bit more sophisticated? Here's the output of `ocramius/package-versions`:

```
Writing lock file
Generating autoload files
ocramius/package-versions:  Generating version class...
ocramius/package-versions: ...done generating version class
```

Reply:
I have added the loaded packages on verbose output (see df75781c37d41035c098fa910a9818ca27527e16)